src/javascript.js

```diff
@@ -284,10 +284,16 @@ console.log(arry[1])
 
 
 // what is the index of 2 ? : 
+//it is the index of (1)
 
 // ==> TASK TWO
 // => explain to me how forEach works 
-
+let arry1 = [1, 2, 3, 4 , 5, 6, 7, 8];
+arry1.forEach(( arr,element, index) => {
+    console.log("arr:", arr, "Index:", index, "element:", element);
+})
```

Comment:
=>good job on task two, 


=>but task one you need to solve it, 
by using indexOf to see what is the index of 2 inside of the array